hack/verify-bazel.sh

```diff
@@ -17,7 +17,8 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-export KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+export KUBE_ROOT
```

Comment:
Do we need to export this at all?

Reply:
unsure. I've been keeping these to non-functional changes where possible. it's hard to tell what actually reads these >.>